Add topic "CSS Framework" (#3901)
github/explore: topics/css-framework/index.md

---
display_name: CSS Framework
short_description: CSS Frameworks usually offer one or more stylesheets with basic styling/generic components, and many of them use preprocessors.
topic: css-framework
related: css, bootstrap, tailwind, zurb-foundation
wikipedia_url: https://en.wikipedia.org/wiki/CSS_framework
---

> **CSS Frameworks** offer one or multiple stylesheets from basic styling to generic components to some extend, including but not limited to

Reviewer: ```suggestion
**CSS Frameworks** offer one or multiple stylesheets from basic styling to generic components to some extend, including but not limited to
```

Can we iterate on the wording here, I'm not sure I understand "to some extend"

Author: It's a typo. I actually intended to write "exten**t**"

* Reboot (aka "Preflight")
* Layout (often times "Grid")
* Forms, Tables, etc.
* [Typography](https://github.com/topics/typography)
* [Responsiveness](https://github.com/topics/responsive) (often times including breakpoints and [viewport](https://github.com/topics/viewport))

> They often times have their own class notation and many of them use [preprocessors](https://github.com/topics/css-preprocessor) like [Sass](https://github.com/topics/sass), [Less](https://github.com/topics/less) oder [Stylus](https://github.com/topics/stylus).

Reviewer: ```suggestion
They often have their own class notation and many use [preprocessors](https://github.com/topics/css-preprocessor) like [Sass](https://github.com/topics/sass), [Less](https://github.com/topics/less) or [Stylus](https://github.com/topics/stylus).
```

Small suggestions on rewording and a fix for what I believe is a typo

Author: It is&mdash;embarrassing German auto correct 😅
